app/build.gradle

```diff
@@ -13,7 +13,7 @@ if (enable_plugin_kotlin) {
     apply plugin: 'kotlin-kapt'
 }
 
-rootProject.ext.version_minSdk = 16
+rootProject.ext.version_minSdk = 18
```

Comment:
There are a bunch of

` if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) { `

in our codebase. May be good to remove them if we are bumping versions